code/game/gamemodes/game_mode.dm

```diff
@@ -112,6 +112,11 @@ GLOBAL_VAR_INIT(cas_tracking_id_increment, 0) //this var used to assign unique t
 	log_game("Server IP: [world.internet_address]:[world.port]")
 	return TRUE
 
+/datum/game_mode/proc/adjust_ammo_values()
+	for(var/ammo in GLOB.ammo_list)
+		if(MODE_HAS_FLAG(MODE_FACTION_CLASH))
+			GLOB.ammo_list[ammo].setup_faction_clash_values()
```

Comment:
```suggestion
	if(!MODE_HAS_FLAG(MODE_FACTION_CLASH))
		return

	for(var/ammo in GLOB.ammo_list)
			GLOB.ammo_list[ammo].setup_faction_clash_values()
```

Reply:
welp I want to make it expandable in soem way to be usable with other gamemode changes, for exaple if UPP had gamemdoe to fight xenos to change up their values

Reply:
but jeh I needs some improvemnt